src/frontend/screens/Settings/components/PlaytimeSync.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ToggleSwitch } from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+import SettingsContext from '../SettingsContext'
+
+const PlaytimeSync = () => {
```

Comment:
I understand we only support playtime sync for GOG games, right? Maybe that should be clarified with an info box component or something.

Also, I wonder if this setting would be better to be per game and not global (like if in the future you want to not sync one specific game). I can imagine users will want that and I think the change is not to much to adapt this PR to that? Just an idea though, not a blocker.